tests/web/mock_web_api_server.py

```diff
@@ -102,11 +104,23 @@ def _handle(self):
 
                 if pattern == "html_response":
                     self.send_response(404)
-                    self.set_common_headers()
+                    self.send_header("content-type", "text/html;charset=utf-8")
```

Comment:
I've changed this content-type that had been unintentionally `application/json; charset=utf-8` from the beginning.